### PR TITLE
Update test/ for openSUSE

### PR DESCRIPTION
**Pull request by fbuihuu**

### CODE REVIEW

File: test/test-functions [hunk 1]
```diff
@@ -1159,6 +1159,7 @@ install_suse_systemd() {
         systemd-coredump
         systemd-experimental
         systemd-journal-remote
+        systemd-network
```
Comment (yuwata) [inline test/test-functions:1165]: The commit message says resolved, but actually networkd.

**Review by yuwata — APPROVED** [review]
LGTM, except for one nit.

File: test/test-functions (hunk 1 above)
Comment (fbuihuu) [reply test/test-functions:1165]: On openSUSE, systemd-network (sub)package includes both networkd and resolved so installing this package should be correct... or maybe I'm missing your point ?

Comment (yuwata) [reply test/test-functions:1165]: ah, i see. then pleaae add a comment about that?

**Comment by fbuihuu** [comment]
@yuwata I added a commit to fix the default timeout values in README.testsuite.

**Comment by bluca** [comment]
```
+ make -C TEST-35-LOGIN setup run
TEST-35-LOGIN SETUP: Tests for systemd-logind
Reusing existing cached image /tmp/autopkgtest.Cw3Y96/build.UxL/systemd/test/TEST-35-LOGIN/../default.img → /tmp/autopkgtest.Cw3Y96/build.UxL/systemd/test/default.img
'/tmp/autopkgtest.Cw3Y96/build.UxL/systemd/test/default.img' -> '/var/tmp/systemd-test.vi7oeF/default.img'
I: Masking supporting services
'/var/tmp/systemd-test.vi7oeF/root/etc/systemd/system/systemd-hwdb-update.service' -> '/dev/null'
'/var/tmp/systemd-test.vi7oeF/root/etc/systemd/system/systemd-journal-catalog-update.service' -> '/dev/null'
'/var/tmp/systemd-test.vi7oeF/root/etc/systemd/system/systemd-networkd.service' -> '/dev/null'
'/var/tmp/systemd-test.vi7oeF/root/etc/systemd/system/systemd-networkd.socket' -> '/dev/null'
'/var/tmp/systemd-test.vi7oeF/root/etc/systemd/system/systemd-resolved.service' -> '/dev/null'
W: Failed to install '/tmp/autopkgtest.Cw3Y96/build.UxL/systemd/manual/test-session-properties'
F: Failed to install /tmp/autopkgtest.Cw3Y96/build.UxL/systemd/manual/test-session-properties
```

**Comment by yuwata** [comment]
(Please backport test-kernel-install fix for v252-stable.)

**Review by yuwata — APPROVED** [review]

**Pull request merged**